Package.swift

```diff
@@ -165,7 +165,7 @@ let package = Package(
         .target(
             /** Package model conventions and loading support */
             name: "PackageLoading",
-            dependencies: ["SwiftToolsSupport-auto", "Basics", "PackageModel", "SourceControl"]),
+            dependencies: ["SwiftToolsSupport-auto", "Basics", "PackageDescription", "PackageModel", "SourceControl"]),
```

Comment:
I think this may cause some problems, since the public declarations in PackageDescription are geared toward use by the manifest.  So anything made public here for use by libSwiftPM would also become public to the manfiest, which may not be what we want.

That said, we do want to avoid duplicating code.  Perhaps there should be another module that both PackageDescription and PackageModel use that contains the common parts?

Reply:
Sounds good. I created ContextModel today. Just let me know which module to put it in. This follows the pattern we discussed.

Reply:
I believe not.